common/src/main/scala/explore/model/Conditions.scala

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats._
+import cats.implicits._
+import monocle.macros.Lenses
+import enum._
+
+@Lenses
+final case class Conditions(
```

Comment:
These should be available in gsp-core

Reply:
Yes, I agree that we should move them to `gsp-core` and share it with `ocs3`. I've just copied them here temporarily.